CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# 4.4.1 (2021-03-19)
+
+- Replaced theme conditionals in templates with the new `skyIfTheme` directive. [#138](https://github.com/blackbaud/skyux-popovers/pull/138)
+- Fixed the dropdown when items are added before the menu is fully open. [#137](https://github.com/blackbaud/skyux-popovers/issues/137)
```

Comment:
```suggestion
- Fixed the dropdown component to prevent an error when items are added before the menu is fully open. [#137](https://github.com/blackbaud/skyux-popovers/issues/137)
```